docs/src/index.md

```diff
@@ -0,0 +1,14 @@
+# Element Array Electrophysiology
+
+This Element features DataJoint schemas for analyzing extracellular array electrophysiology data acquired with Neuropixels probes and spike sorted using Kilosort spike sorter. Each Element is a modular pipeline for data storage and processing with corresponding database tables that can be combined with other Elements to assemble a fully functional pipeline.
+
+The Element is comprised of `probe` and `ephys` schemas. There are several `ephys` schemas including `ephys_acute`, `ephys_chronic`, and `ephys_precluster` to handle several use cases of this pipeline and workflow. Visit the [Concepts page](./concepts.md) for more information about the use cases of `ephys` schemas and an explanation of the tables. To get started with building your own data pipeline, visit the [Tutorials page](./tutorials.md).  
+
+### `ephys_acute` module
```

Comment:
I think we can add a oneliner about each - This was in the original readme. And then the concepts page should at least mention them by name. Looks like we're missing mention of `no-curation` which will be important to have, as that's on Works Free tier

Reply:
Added mention of `no-curation` and a short description of each